server/src/test/java/io/crate/integrationtests/DynamicMappingUpdateITest.java

```diff
@@ -213,17 +202,6 @@ private void execute_alter_table_add_column_results_in_dynamic_mapping_updates()
             o['q']| 7
             o['q']['r']| 8
             o['q']['r']['s']| 9
-            """,
-            """
-            id| 1
-            name| 2
-            o| 3
-            o['b']| 4
-            o['a']| 5
-            o['a']['b']| 6
-            o['q']| 7
-            o['q']['r']| 8
-            o['q']['r']['s']| 9
```

Comment:
I must have assumed that `ALTER` suffers from the same column order issue.
But `ALTER` actually goes through `TableElementAnalyzer` which adds column positions so that it does not depend on the insertion order unlike `UPDATE`.